pkg/agent/config/node_config.go

```diff
@@ -204,6 +200,8 @@ type NetworkConfig struct {
 	TransportIfaceCIDRs   []string
 	IPv4Enabled           bool
 	IPv6Enabled           bool
+	MTUDeduction          int
+	InterfaceMTU          int
```

Comment:
Please copy the original comments before "NodeMTU" here.

```
	// Set either via defaultMTU config in antrea.yaml or auto discovered.
	// Auto discovery will use MTU value of the Node's primary interface.
	// For Encap and Hybrid mode, Node MTU will be adjusted to account for
        // encap header.
```

Maybe change " primary interface" to "transport interface".

Reply:
Done